src/instruments/src/EFB/SimbriefApi/simbriefInterface.ts

```diff
@@ -16,6 +16,8 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+/* eslint-disable camelcase */
```

Comment:
I'd be more inclined to just rename this and drop the hungarian notation, but I see why this PR itself should maybe not do that and rather just bypass the rule.

Reply:
this is parsing data from an API